cluster/envoy/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # =============================
 FROM golang:1.8-alpine as build
 # =============================
-RUN apk --no-cache add git
+RUN apk --no-cache add git gcc musl-dev
```

Comment:
For what it's worth, you can also just use [build-base](https://pkgs.alpinelinux.org/package/v3.3/main/x86/build-base) as a good general base package (equivalent to build-essential) when you want to ensure you have a good base toolchain without thinking about each specific build tool.